cmd/antrea-agent/options.go

```diff
@@ -758,3 +762,52 @@ func (o *Options) validateNodePortLocalConfig() error {
 	}
 	return nil
 }
+
+func validateNodeNetworkPolicyPrivilegedRules(rules []agentconfig.PrivilegedRule) error {
+	if len(rules) == 0 {
+		return fmt.Errorf("no privileged rule is found. The absence of privileged rules may result in network unavailability in the Kubernetes cluster")
+	}
+	regex := regexp.MustCompile(`^\d+(?::\d+)?$`)
+
+	for _, rule := range rules {
+		protocol := strings.ToLower(rule.Protocol)
+		switch protocol {
+		case "tcp":
+			fallthrough
+		case "udp":
+			fallthrough
+		case "sctp":
```

Comment:
ditto, it only needs to ensure re-applying policies can work, no point to support irrevalent protocols.

Reply:
Will remove this protocol.